test/models/vote_test.rb

```diff
@@ -0,0 +1,50 @@
+require "test_helper"
+
+describe Vote do
+  before do
+    @user = users(:brock)
+    @pizza = pizzas(:pepperoni)
+    @vote = Vote.new(user: @user, pizza: @pizza)
+  end
+
+  describe "validations" do
```

Comment:
You should also have a validation which ensure that the combination of user_id and pizza_id must be unique.